src/bin/arvo.rs

```diff
@@ -0,0 +1,61 @@
+#![feature(convert)]
```

Comment:
This was just to get access to the `to_cstring()` conversion helper, we can just manually construct a `CString` if we want to remain on stable.